src/components/reusable/pagetitle/pageTitle.scss

```diff
@@ -31,7 +33,7 @@
 
   &-subtitle {
     @include typography.type-body-02;
-    color: var(--kd-color-text-secondary);
+    color: var(--kd-color-text-title-secondary);
```

Comment:
I wasn't able to verify if this token is correct in figma. I would double-check with the designers since it matches the secondary title token. Same for the icon color.

Reply:
@brian-patrick-3 ,
Changed color for subtitle:
    color: var(--kd-color-text-level-secondary);

Changed icon to duotone